src/coreclr/jit/valuenum.h

```diff
@@ -732,8 +732,12 @@ class ValueNumStore
                                bool         srcIsUnsigned    = false,
                                bool         hasOverflowCheck = false);
 
+    // Compute the ValueNumber for a bitcast
     ValueNum VNForBitCast(ValueNum srcVN, var_types castToType);
 
+    // Compute the ValueNumberPair for a bitcast
+    ValueNumPair VNPairForBitCast(ValueNumPair srcVNPair, var_types castToType);
```

Comment:
```suggestion
    ValueNum VNForBitCast(ValueNum srcVN, var_types castToType);

    ValueNumPair VNPairForBitCast(ValueNumPair srcVNPair, var_types castToType);
```
We're moving away from non-header comments in `valuenum.h`.

Reply:
Fixed.